servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -171,6 +172,17 @@ public Inode getInodeByPath(TachyonURI path) throws InvalidPathException {
   }
 
   /**
+   * @param path the path to get the inodes list for
+   * @return the inodes list with the given path
+   * @throws InvalidPathException if the path is invalid
+   */
+  public List<Inode> collectInodes(TachyonURI path) throws InvalidPathException {
```

Comment:
Could you rename it to `getInodeAncestorsRecursive`, similar to `getInodeChildrenRecursive`?